configschema.json

```
@@ -307,6 +307,17 @@
         }
       },
       "required": ["enabled"]
+    },
+    "tiltify": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "enable": { "type": "boolean", "default": true },
```

```suggestion
        "enable": { "type": "boolean", "default": false },
```

Would it not be better to have this off by default?

---

Yes, ty, that was a leftover